src/main/java/mods/eln/sixnode/EmergencyLamp.kt

```diff
@@ -167,6 +169,8 @@ class EmergencyLampElement(sixNode: SixNode, side: Direction, descriptor: SixNod
         electricalComponentList.add(chargingResistor)
         slowProcessList.add(process)
         slowProcessList.add(NodePeriodicPublishProcess(sixNode, 2.0, 0.5))
+        slowProcessList.add(VoltageStateWatchDog().set(load).setUNominal(desc.cable.electricalNominalVoltage)
```

Comment:
The resistance is constant for each model and taking I = U/R and P = U^2/R into account we see that only voltage is actually significant in this case.